eduNEXT/xblock-filesmanager
# feat: rename folders and remove hidden files button

### Description

This allows you to rename a folder in the tree of directories and remove the 'hidden files' from the toolbar.

![ezgif com-video-to-gif (3)](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/5825f4f9-afc9-44fb-9e4b-ac06f8455e35)


#### "Show hidden files" button

Before           |  Now
:-------------------------:|:-------------------------:
![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/10f57a8a-3426-4a28-8500-cd858e36d477)  |  ![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/0c4909dd-22e6-45ac-9620-c4426d1b3c39)

### How to test it

- Check this branch`jv/fix-chonky-improvements` in the XBlock files manager

1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 
```
------------------------------------------------------------------------------------------------------------------------------------------------
Now select a folder to rename (click in the folder) -> "Rename folder"

![rename-folder](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/29f85845-7ce7-43e8-8e69-2cb9011b556f)

You will see a prompt to change the name of the folder

![rename-prompt](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/371940d1-a02e-488e-9597-a8e8b2488ebc)



To check **"Show hidden files"** button, you would need to click **"Options"** button

![optiones-button](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/b8b5c9a8-0202-492e-acc5-7aaf1548af1c)



**Merge checklist:**
Check off if complete *or* not applicable:
